src/test/java/org/sagebionetworks/common/util/progress/PathNormalizerTest.java

```diff
@@ -0,0 +1,68 @@
+package org.sagebionetworks.common.util.progress;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.sagebionetworks.common.util.PathNormalizer.normalizeMethodSignature;
+
+public class PathNormalizerTest {
+
+	/*
+	 * Synapse API Tests
+	 */
+	@Test
+	public void urlWithoutParameters() {
+		assertEquals("/certifiedusertestresponse", normalizeMethodSignature("/repo/v1/certifiedUserTestResponse", "POST"));
+	}
+
```

Comment:
missing all three test cases where the url does not start with /repo/v1 or /auth/v1 or /file/v1